src/content/app/genome-browser/components/browser-cog/BrowserCog.tsx

```diff
@@ -65,16 +65,16 @@ const BrowserCog = (props: BrowserCogProps) => {
     icon: CogIcon
   };
 
-  const [showTrackConfig, setTrackConfigAnimation] = useState(cogActivated);
+  const [showTrackSettings, setTrackSettingsAnimation] = useState(cogActivated);
```

Comment:
I'm surprised by the name of the setter function — `setTrackSettingsAnimation` (previously, `setTrackConfigAnimation`). Looks like its purpose is only to set a boolean flag for whether the track settings panel is displayed or not.